File: JavaScript Fundamentals/FunctionsHomework/Problem 2. Reverse number/ReverseNumber.js
var arr = ['256', '123.45', '-876', '7331'],
    i,
    len;

for (i = 0, len = arr.length; i < len; i += 1) {
    console.log(reverseNumber(arr[i]));
}

function reverseNumber(number) {
    var j,
        result = '';
    for (j = number.length - 1; j >= 0; j -= 1) {
        if (number[j] === '-') {
            result = '-' + result;
        }
        else {
            result += number[j];
        }
    }
    return result;
}

File: CSharp Part 2/MethodsHomework/Problem 07. Reverse number/ReverseNumber.cs
// Problem 7. Reverse number
// Write a method that reverses the digits of given decimal number.

using System;

class ReverseNumber
{
    static void Main()
    {
        Console.WriteLine("This program reverses the digits of given decimal number.");
        Console.Write("Please enter a decimal number: ");
        string number = Console.ReadLine();

        ReversedNumber(number);
    }

    static void ReversedNumber(string number)
    {
        Console.Write("Result: ");
        for (int i = number.Length - 1; i >= 0; i--)
        {
            Console.Write(number[i]);
        }
        Console.WriteLine();
    }
}